object/service.proto

```diff
@@ -84,6 +84,8 @@ message PutRequest {
         Object Object       = 1;
         // Token with session public key and user's signature
         session.Token Token = 2;
+        // Number of the object copies to store within the RPC call (zero is processed according to the placement rules)
+        uint32 CopiesNumber = 3;
```

Comment:
mb we can be named it like in the container service? `ReplFactor`

Reply:
The meaning is different here. Technically it's a number of ACKs from Storage nodes. Let it be Copies here to simplify understanding.

Reply:
thanks for explaining

Reply:
@im-kulikov, explanations from @realloc were ahead of mine.
@realloc, clarify, please, do you agree w/ `CopiesNumber` or propose to trim it to `Copies`.

Reply:
`CopiesNumber` is fine